pyproject.toml

```diff
@@ -0,0 +1,64 @@
+[tool.poetry]
+name = "clickhouse-sqlalchemy"
+version = "0.3.2"
+description = "SQLAlchemy dialect for ClickHouse"
+authors = ["Konstantin Lebedev <kostyan.lebedev@gmail.com>"]
+license = "MIT"
+readme = "README.rst"
+include = ["CHANGELOG.md"]
+exclude = ["tests"]
+repository = "https://github.com/xzkostyan/clickhouse-sqlalchemy"
+documentation = "https://clickhouse-sqlalchemy.readthedocs.io/en/latest/"
+keywords = ["clickhouse", "sqlalchemy"]
+classifiers = [
+    "Development Status :: 4 - Beta",
+    'Environment :: Console',
+    'Intended Audience :: Developers',
+    'Intended Audience :: Information Technology',
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+    "Programming Language :: SQL",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Topic :: Database",
+    "Topic :: Software Development",
+    "Topic :: Software Development :: Libraries",
+    "Topic :: Software Development :: Libraries :: Application Frameworks",
+    "Topic :: Software Development :: Libraries :: Python Modules",
+    "Topic :: Scientific/Engineering :: Information Analysis",
+]
+
+[tool.project.urls]
+"PyPI" = "https://pypi.org/project/clickhouse-sqlalchemy/"
+"Changes" = "https://github.com/xzkostyan/clickhouse-sqlalchemy/blob/master/CHANGELOG.md"
+
+[tool.poetry.dependencies]
+python = ">=3.9,<4.0"
+alembic = "^1.13.3"
+asynch = "0.2.3"
+clickhouse-driver = ">=0.1.2"
+requests = "^2.32.3"
+sqlalchemy = "^2.0.36"
+
+[tool.poetry.group.test.dependencies]
+pytest = "^7.4"
+pytest-asyncio = "^0.23.0"
+parameterized = "^0.9.0"
+responses = "^0.25.3"
+
+[tool.pytest.ini_options]
+minversion = "7.4"
+asyncio_mode = "auto"
+
+[tool.poetry.plugins."sqlalchemy.dialects"]
```

Comment:
https://python-poetry.org/docs/pyproject#plugins - 1.8
https://python-poetry.org/docs/main/pyproject#entry-points - main

The versions differ, now it works with the deprecated in future syntax -> `tool.poetry.plugins` -> will be `project.entry-points.`